config/settings.yml

```diff
@@ -137,6 +137,10 @@ sys_api_key:
   default: ''
 repository_authentication_caching_enabled:
   default: 1
+repositories_automatic_managed_repos:
+  default: 0
```

Comment:
Would you mind adding a `type: bool`, even though this has [no effect](https://github.com/opf/openproject/pull/3270) today?